doc/nrf/ug_nrf7002_constrained.rst

```diff
@@ -0,0 +1,137 @@
+.. _nRF7002dk_nRF5340_constrained_host:
+
+Operating with a resource constrained host
+##########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide provides recommendations and guidelines for using the nRF7002 as a companion chip on resource-constrained hosts such as the nRF5340 SoC.
+
+Zephyr OS factors
+*****************
+
```

Comment:
Please provide an introductory statement for Zephyr OS factors. The section cannot be left blank without any content.